docs/TPC-H.md

```diff
@@ -6,7 +6,8 @@ After the installation of [Pixels + Trino](INSTALL.md), we can evaluate the perf
 
 Attach a volume that is larger than the scale factor (e.g., 150GB for SF100) to the EC2 instance.
 Mount the attached volume to a local path (e.g., `/data/tpch`).
-Download tpch-dbgen to the instance, build it, and generate the dataset and queries into the attached volume.
+Download `tpch-dbgen` to the instance, build it, and generate the dataset and queries into the attached volume.
+You can follow [this article](https://docs.deistercloud.com/content/Databases.30/TPCH%20Benchmark.90/Data%20generation%20tool.30.xml?embedded=true) to correctly use `tpch-dbgen` for dataset generation.
```

Comment:
No need to reference this link, since the readme of tpch-dbgen already explains how to use it clearly.